agent/lib/src/utils.dart

```diff
@@ -257,7 +257,9 @@ Future<Null> getFlutterAt(String revision) async {
 
 Future<Process> startProcess(String executable, List<String> arguments,
     {Map<String, String> env, bool silent: false}) async {
-  String command = '$executable ${arguments?.join(" ") ?? ""}';
+  final List<String> logSafeArguments = arguments ?? <String>[];
+  logSafeArguments.removeWhere((String arg) => arg.contains('cloud-auth-token'));
```

Comment:
maybe create a const set excludeList in case we need to add more?

Reply:
Done